src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculetion {
+    private double radius;
+
+    public Circle(double radius, String color, String name) {

```

Comment:
The constructor of `Circle` should initialize all fields of the superclass `Figure` by calling `super(color, name)`.

Reply:
fix comment